Src/Base/AMReX.cpp

```diff
@@ -148,15 +148,13 @@ amrex::write_to_stderr_without_buffering (const char* str)
     }
 }
 
+#if !AMREX_DEVICE_COMPILE
 namespace {
 //  Having both host and device versions to avoid compiler warning
 AMREX_GPU_HOST_DEVICE
```

Comment:
Seems that we can remove these 2 lines now.  It was added for HIP.  But with the new way, it's not needed anymore.

Reply:
@WeiqunZhang removed the two lines in 649cf41